conf/modules.config

```diff
@@ -404,7 +404,7 @@ process {
         ]
         ext.args = ['outname':'', 'model':'hierarchical',
                     'method':'nt', 'linkage':'single',
-                    'skip_overlap':true,
+                    'skip_convergence':false,
```

Comment:
As this still takes really long to compute,  I would be partidary of leaving it true by default and if the user wants to compute it they can activate it

Reply:
```suggestion
                    'skip_convergence':true,
```